routes/web.php

```diff
@@ -16,7 +16,7 @@
 
 /*
 |-------------------------------------
-| Main Site Admin Route Group
+| Main Site Admin Route GroupΩ
```

Comment:
Why with the ORM?

Reply:
that is mistakenly typed, i just noticed